### PR TITLE
Reformat .md files

### PR DESCRIPTION
**Pull request by st-pasha**
Trying to eliminate warning messages from RemarkLint plugin.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -20,50 +21,87 @@ This document is a set of guidelines for contributors, nothing is set in stone s
...
 ### Enhancements and Feature requests
 
 We are open to new enhancements/feature requests!
 
-* Just like with bugs, please check whether there is already an existing issue in the tracker.
-* If not, then you can reach create a new issue with an appropriate label (`feature request` or `enhancement`).
+* Just like with bugs, please check whether there is already an existing issue
+  in the tracker.
+
+* If not, then you can reach create a new issue with an appropriate label
```
Comment (oleksiyskononenko) [inline .github/CONTRIBUTING.md]: Shouldn't it be just `then you can create`?

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -4,18 +4,20 @@ about: Is something not right? Let us know.
...
 
-  - Your environment?
-    What is your datatable version, python version, and operating system?
+- Your environment?
+  What is your datatable version, python version, and operating system?
 
-  - Tag the issue with `[bug]` or `[segfault]` (depending on whether it crashes Python or not)
-    Thank you for contributing, and sorry for any inconvenience...
+- Tag the issue with `[bug]` or `[segfault]` (depending on whether it crashes
+  Python or not). Thank you for contributing, and sorry for the inconvenience...
```
Comment (angela0xdata) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: >Thank you for contributing, and sorry for the inconvenience...
Should that line be part of the "tag the issue" bullet? It seems like it should be outside of the steps. Also, you can remove the ellipses and just end with a period.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (st-pasha) [reply .github/CONTRIBUTING.md]: 👍

File: .github/ISSUE_TEMPLATE/Bug_report.md (hunk 2 above)
Comment (st-pasha) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: I moved that sentence into a new line.

**Comment by angela0xdata** [comment]
Made some minor edits directly in the branch.

**Pull request merged**